pkg/requestcontext/context.go

```diff
@@ -92,3 +92,29 @@ func (acCtx *AccessControlContext) matchesRepo(globPatterns map[string]bool, rep
 
 	return allowed
 }
+
+// request-local context key.
+var authnMdwCtxKey = Key(1) //nolint: gochecknoglobals
```

Comment:
s/mdw/mdware 

Would rather pick a well-known acronym here, look at gorilla-mux project etc

Reply:
or traefik project

Reply:
they use amw for authn middleware, as short form and authenticationMiddleware as long form